Rakefile

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env rake
+# frozen_string_literal: true
+
 require 'bundler/setup'
-require "bundler/gem_tasks"
+require 'bundler/gem_tasks'
 
 begin
   require 'rspec/core/rake_task'
 
   RSpec::Core::RakeTask.new(:spec)
 
-  task :default => :spec
-rescue LoadError
+  task default: :spec
+rescue LoadError # rubocop:disable Lint/SuppressedException
 end
```

Comment:
This begin...rescue block shouldn't be necessary. Possibly a vestige of a much older way of loading gem dependencies.